CodeStarterCamp_Week2/main.swift

```diff
@@ -11,6 +11,19 @@ import Foundation
 var lottoRound: [Int: [Int]] = [:]
 
 
+var savedLottoRounds: [Int: [Int]] = [:]
+var round = 1
+
+
+func changeToStr(customStr: [Int]) -> String {
+    var str = customStr.description
+    str.removeFirst()
+    str.removeLast()
+    
+    return str
+}
```

Comment:
`change`라는 동사로 네이밍을 시작한 것은 좋습니다만 [Naming](https://www.swift.org/documentation/api-design-guidelines/#naming) 파트를 전반적으로 다시 읽어보시는 것이 좋겠습니다.

[Promote Clear Usage](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage) 파트와 [Use Terminology Well](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well) 파트입니다.

우선 내부 변수 및 함수명에 쓰이는 `str`은 줄임말이면서 `String` 타입을 의미합니다.
- Name variables, parameters, and associated types according to their roles, rather than their type constraints.
- Avoid abbreviations.

파라미터명인 `customStr` 또한 의미가 명확하지 않습니다.

`changeToText(from numbers: [Int]) -> String` 정도의 네이밍이었으면 좋았겠다고 생각합니다.